api/v1beta1/zz_generated.deepcopy.go

```diff
@@ -1,3 +1,4 @@
+//go:build !ignore_autogenerated
```

Comment:
This is due to go 1.17, and will get auto_generated for anyone on that go version who compiles